src/components/slider/SliderThumb.vue

```diff
@@ -222,6 +222,14 @@ export default {
                 this.oldValue = value
             }
         }
+    },
+
+    beforeDestroy() {
+        document.removeEventListener('mousemove', this.onDragging)
+        document.removeEventListener('touchmove', this.onDragging)
+        document.removeEventListener('mouseup', this.onDragEnd)
+        document.removeEventListener('touchend', this.onDragEnd)
+        document.removeEventListener('contextmenu', this.onDragEnd)
```

Comment:
I do not see any problem with the original code. But there might be some edge cases I overlooked. So it will be safer to have these lines here.